### DISCUSSION TITLE
Does Delve support freezing and thawing threads / goroutines?

### DISCUSSION THREAD

**Discussion by alicia-c** [go-delve/delve#3096] [Q&A] [answered]
For complex multi-threaded applications, it is extremely useful to be able to freeze and thaw threads / goroutines in order to reproduce race conditions. Was wondering if Delve supports this or has any plans to support this.

Thanks!

**Comment by aarzilli** [comment, marked as answer]
No, it can not be done without runtime assistence, see #1529 and related.

**Reply by alicia-c** [reply]
Ah, got it. Thank you for the response!
